pallets/connectors-gateway/axelar-gateway-precompile/src/lib.rs

```diff
@@ -0,0 +1,397 @@
+// Copyright 2021 Centrifuge Foundation (centrifuge.io).
```

Comment:
@mustermeiszer do you think we may have more precompiles in the future within connectors? I see that Moonbeam has a whole `precompiles` directory at the root of their repo. I don't think we need that now but maybe having a `precompiles` dir here now containing the `axelar-gateway` precompile would make sense.

not a blocker whatsoever tho!

Reply:
Not sure tbh. Would keep it like that for now and refactor later.